tests/e2e/default/service-worker/service-worker.wdio-spec.js

```diff
@@ -111,34 +116,35 @@ describe('Service worker cache', () => {
     expect(cacheDetails.urls.sort()).to.have.members([
       '/',
       '/audio/alert.mp3',
+      '/deploy-info.json',
       '/extension-libs',
       '/fontawesome-webfont.woff2',
       '/fonts/NotoSans-Bold.ttf',
       '/fonts/NotoSans-Regular.ttf',
       '/fonts/enketo-icons-v2.woff',
       '/img/cht-logo-light.png',
-      '/img/icon.png',
       '/img/icon-chw-selected.svg',
       '/img/icon-chw.svg',
+      '/img/icon-close.svg',
```

Comment:
This list is the same as before, except for this new image that we added as part of the design of the new nav.